### ISSUE TITLE
Normalization

### ISSUE THREAD

**Issue by lunathanael** [lunathanael/racing#6]
Batch normalization, etc.

https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.normalize.html
https://docs.pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html
https://docs.pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html
